tradetrust/open-attestation-api/src/app.js

```diff
@@ -42,20 +43,59 @@ function create(){
     logger.error(err);
   }
 
-  app.post('/document/wrap', async function (req, res){
+  
+  app.post("/document/sign", async function(req,res){
+    console.log('/document/sign')
+    if (req.body.document === undefined){throw new UserFriendlyError('No "document" field in payload');}
+    const document = req.body.document; //should be wrapped already at this point
+
+    var keyPair;
+    if(!process.env.DOCUMENT_STORE_OWNER_PRIVATE_KEY){
+      keyPair = {
+        private: '0x40b639e8fb83afe8ad8d4bb7857e69d039a15c3e476b2d98f346222623420e6f',
+        public: '0x103D912298C89a98Eed323376aa4403b92b28842'
+      }
+    }
+    else{
+      keyPair = {
+        //public: `did:ethr:${publicKey}#controller`,
+        public: process.env.DOCUMENT_STORE_OWNER_PUBLIC_KEY,
```

Comment:
won't the public key need to be wrapped in that string to be a valid did? per the oa docs...

Reply:
Ah yes... you're right. I was thinking that `public` was a public key. But it ends up directly as the `verificationMethod` of the proof in the final doc. So `public` kinda means something from which the public key can be derived, not the key itself.

Any thoughts on whether we can/should use the full public key here vs the ethr address? Both would be valid DIDs. The possible value of using the full key I can see is that verification can be done offline, assuming that the true ethr address of the issuer is already known.